src/main/resources/db/migration/V3__Add_version_code_to_artifact_versions.sql

```diff
@@ -1,2 +1,3 @@
 ALTER TABLE artifacts_versions ADD COLUMN version_code BIGINT NOT NULL;
-ALTER TABLE artifacts_versions ALTER COLUMN filename varchar NOT NULL
\ No newline at end of file
+ALTER TABLE artifacts_versions ALTER COLUMN filename varchar NOT NULL;
```

Comment:
you shouldn't change existing migration because it already applied on production

Reply:
I've changed my new migration to the new file.